osu.Game.Rulesets.Taiko/Mods/TaikoModClassic.cs

```diff
@@ -1,25 +1,67 @@
 // Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
+using System;
 using System.Diagnostics;
+using System.Linq;
+using System.Collections.Generic;
+using osu.Framework.Graphics;
+using osu.Framework.Bindables;
 using osu.Game.Rulesets.Mods;
 using osu.Game.Rulesets.Taiko.Objects;
 using osu.Game.Rulesets.Taiko.UI;
 using osu.Game.Rulesets.UI;
+using osu.Game.Rulesets.Objects.Drawables;
+using osu.Game.Rulesets.Taiko.Objects.Drawables;
 
 namespace osu.Game.Rulesets.Taiko.Mods
 {
-    public class TaikoModClassic : ModClassic, IApplicableToDrawableRuleset<TaikoHitObject>, IUpdatableByPlayfield
+    public class TaikoModClassic : ModClassic, IApplicableToDrawableRuleset<TaikoHitObject>, IApplicableToDrawableHitObject, IUpdatableByPlayfield
     {
+        /// <summary>
+        /// The maximum aspect ratio with classic mod enabled.
+        /// </summary>
+        private const float classic_max_aspect_ratio = 2.0f / 1.0f;
+
+        /// <summary>
+        /// The maximum time range with classic mod enabled. This will be enforced in different ways depending on the
+        /// <see cref="Mod"/>s enabled.
+        /// </summary>
+        private static readonly double classic_max_time_range = DrawableTaikoRuleset.AspectRatioToTimeRange(classic_max_aspect_ratio);
+
+        /// <summary>
+        /// The classic hidden aspect ratio. Note that time rate is also stretched to this from the default aspect ratio.
```

Comment:
```suggestion
        /// The classic hidden aspect ratio. Note that time range is also stretched to this from the default aspect ratio.
```

I assume?

Reply:
Oops, applied in abac44acc938fa90caa53a16ac0af5f172f3a8f7.